Mage/src/main/java/mage/abilities/effects/common/ExileSpellWithTimeCountersEffect.java

```diff
@@ -38,14 +49,17 @@ public ExileSpellWithTimeCountersEffect copy() {
     @Override
     public boolean apply(Game game, Ability source) {
         Player controller = game.getPlayer(source.getControllerId());
-        Card card = game.getStack().getSpell(source.getSourceId()).getCard();
+        Card card = game.getCard(source.getSourceId());
         if (controller == null || card == null) {
-            return true;
+            return false;
         }
         UUID exileId = SuspendAbility.getSuspendExileId(controller.getId(), game);
         if (!card.isCopy() && controller.moveCardsToExile(card, source, game, true, exileId, "Suspended cards of " + controller.getName())) {
             card.addCounters(CounterType.TIME.createInstance(3), source.getControllerId(), source, game);
             game.informPlayers(controller.getLogName() + " exiles " + card.getLogName() + " with " + counters + " time counters on it");
+            if(gainsSuspend){
```

Comment:
style: please match existing codebase, spaces before opening parenthesis and after closing parenthesis